cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -102,19 +102,19 @@
         ('Youth Financial Education', '/educational-resources/youth-financial-education/', [])
     ),(
         ('Resources for Libraries', '/library-resources/', []),
-        ('Resources for Tax Preparers', '/tax-preparer-resources/', []),
+        ('Resources for Tax Preparers', '/educational-resources/resources-for-tax-preparers/', []),
         ('Resources for Parents', '/educational-resources/money-as-you-grow/', [])
     ),(
         ('Information for Economically Vulnerable Consumers', '/empowerment/', []),
         ('Resources for Older Adults and Their Families', '/educational-resources/resources-for-older-adults/', []),
-        ('Free Brochures', 'http://pueblo.gpo.gov/CFPBPubs/CFPBPubs.php', [])
+        ('Free Brochures', 'https://pueblo.gpo.gov/CFPBPubs/CFPBPubs.php', [])
     )]),
```

Comment:
I've been meaning to look into [wagtailmenus](https://github.com/rkhleics/wagtailmenus) which seems designed for this use case.